addons/delivery/models/delivery_carrier.py

```diff
@@ -187,3 +189,16 @@ def fixed_get_tracking_link(self, picking):
 
     def fixed_cancel_shipment(self, pickings):
         raise NotImplementedError()
+
+    def merge_pdf(self, label_name, label_data):
```

Comment:
Also document input variables: we should not have to guess that data is a list of pairs